tests/excerptexport/management/test_purge_old_result_files.py

```diff
@@ -24,6 +24,21 @@ def output_file_with_file_too_old(db, output_file_with_file, mocker):
     return output_file_with_file
 
 
+@pytest.fixture
+def output_file_too_old(db, output_file, mocker):
+    from osmaxx.excerptexport._settings import RESULT_FILE_AVAILABILITY_DURATION
+    just_a_bit_too_old = timezone.now() - RESULT_FILE_AVAILABILITY_DURATION - timezone.timedelta(minutes=1)
+
+    output_file.file_removal_at = just_a_bit_too_old
+    output_file.save()
+
+    output_file.refresh_from_db()
+
+    assert output_file.file_removal_at == just_a_bit_too_old
```

Comment:
Using `refresh_from_db()` I want to be sure that my `file_removal_at` is correct in the database, if we change the code (I did while working on it), the error is very hard to discover.